addons/account/views/account_move_views.xml

```diff
@@ -1063,32 +1064,42 @@
                                            groups="account.group_account_readonly"
                                            options="{'no_create': True, 'no_open': True}"
                                            readonly="posted_before and name not in (False, '', '/')"/>
-
-                                    <span class="o_form_label mx-3 oe_edit_only"
-                                          groups="account.group_account_readonly"
-                                          invisible="move_type == 'entry'">
-                                        <span groups="base.group_multi_currency">in </span>
```

Comment:
It's a lot better, but we lost an alignment, can't we align everything?
![image](https://github.com/user-attachments/assets/49c0c952-cd66-4317-a8c5-c39645003dee)